.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+  },
+  extends: ['google', 'plugin:prettier/recommended'],
```

Comment:
is it intentional to use different linting config/plugins here than on the UI? see [the other repos eslintrc in comparison](https://github.com/WoWAnalyzer/WoWAnalyzer/blob/shadowlands/.eslintrc.js#L4)

additionally, eslint-config-google is anything but maintained, [having had no release in over 2 years](https://www.npmjs.com/package/eslint-config-google)